website/app.js

```diff
@@ -4,6 +4,17 @@ export async function getJson(path) {
   return (await fetch(path)).json();
 }
 
+export function getTravisData() {
+  const url = "https://api.travis-ci.com/repos/denoland/deno/builds?event_type=pull_request#number=20";
+  return fetch(url, {
+    headers: {
+      "Accept": "application/vnd.travis-ci.2.1+json"
+    }
+  })
+  .then(res => res.json())
+  .then(data => data.builds.reverse());
+}
```

Comment:
We should have a way of testing this function... At minimum call it from app_test.js - but it might make sense to download an example of the travis response into the repo and test the parsing on it... What do you think?

Reply:
Do you mean move this function to app_test.js?